### PR TITLE
Create ui-c8y-1020-14-7-preserve-white-spaces-and-new-lines-in-shell-component.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [DM-3840](https://cumulocity.atlassian.net/browse/DM-3840)
Original PR: [6802](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6802)

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1020-14-7-preserve-white-spaces-and-new-lines-in-shell-component.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Preserve white spaces and new lines in shell component
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-14-7-preserve-white-spaces-and-new-lines-in-shell-component.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Shell component preserves whitespace and new lines
```

File: content/change-logs/device-management/ui-c8y-1020-14-7-preserve-white-spaces-and-new-lines-in-shell-component.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3840
+version: 1020.14.7
+---
+Preserve white spaces and new lines in shell component
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1020-14-7-preserve-white-spaces-and-new-lines-in-shell-component.md:17]: The Claude AI suggested to use the following body:
```suggestion
The shell component is commonly used to display code snippets or command line output. Previously, the shell component stripped out whitespace and new lines which could lead to incorrectly formatted or hard to read content. With this change, the shell component now preserves all whitespace and new line characters allowing code snippets and command line output to be properly formatted. This improves the readability of content displayed in the shell component for end users.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/device-management/ui-c8y-1020-14-7-preserve-white-spaces-and-new-lines-in-shell-component.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3840
+version: 1020.14.7
+---
+The shell component is commonly used to display code snippets or command line output. Previously, the shell component stripped out whitespace and new lines which could lead to incorrectly formatted or hard to read content. With this change, the shell component now preserves all whitespace and new line characters allowing code snippets and command line output to be properly formatted. This improves the readability of content displayed in the shell component for end users.
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-14-7-preserve-white-spaces-and-new-lines-in-shell-component.md:17]:
```suggestion
The shell component is commonly used to display code snippets or command line output. Previously, the shell component stripped out whitespaces and new lines which could lead to incorrectly formatted or hard to read content. With this change, the shell component now preserves all whitespaces and new line characters allowing code snippets and command line output to be properly formatted. This improves the readability of content displayed in the shell component for users.
```

File: content/change-logs/device-management/ui-c8y-1020-14-7-preserve-white-spaces-and-new-lines-in-shell-component.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Shell component preserves whitespace and new lines
```
Comment (BeateRixen) [inline content/change-logs/device-management/ui-c8y-1020-14-7-preserve-white-spaces-and-new-lines-in-shell-component.md:3]:
```suggestion
title: Shell component preserves whitespaces and new lines
```

**Review by BeateRixen — APPROVED** [review]

**Review by dafkk — APPROVED** [review]

**Pull request merged**